README.md

```diff
@@ -40,6 +40,18 @@ Ensure you have the [vagrant-disksize](https://github.com/sprotheroe/vagrant-dis
 $ vagrant plugin install vagrant-disksize
 ```
 
+Starting with Virtualbox 6.1.28, [host-only networks](https://www.virtualbox.org/manual/ch06.html#network_hostonly) are restricted to `192.168.56.0/21` by default. 
+
+We will need to do the following to override this restriction:
+
+```bash
+mkdir /etc/vbox
+
+echo "
+* 192.168.56.0/21
+* 33.33.0.0/16" >> /etc/vbox/networks.conf
```

Comment:
```suggestion
* 33.33.0.0/16" | sudo tee /etc/vbox/networks.conf
```

Reply:
I committed the diff, so this should be all set